### ISSUE TITLE
Indicator for missing alt text

### ISSUE THREAD

**Issue by deanveloper** [sk22/megalodon#355]
**Is your feature request related to a problem? Please describe.**
A clear and concise description of what the problem is. 

I get very frustrated when I boost media which doesn't have alt-text, and when I accidentally forget to include alt-text when I'm making a post.

**Describe the solution you'd like**

Glitch-soc has a nice local setting which adds a confirmation dialog when you post/boost media which doesn't have alt-text, which is nice because I sometimes forget to add it. It would be great if there could be a setting in Megalodon to add a confirmation dialog when posting/boosting altless media

**Describe alternatives you've considered**

Other issues point out maybe adding an icon to images without alt-text, but this doesn't help accidentally posting without alt-text. I also use [@PleaseCaption@botsin.space](https://botsin.space/@PleaseCaption) which is very useful, but ideally I'd like it if I never made the post/boost without alt-text in the first place.

**Additional context**

For context, here is what the glitch-soc dialog looks like:

![Screenshot_20230122-124554](https://user-images.githubusercontent.com/3196327/213934019-2498682f-9d02-486c-8952-5e3ab80acdeb.png)

**Comment by sk22** [comment]
![Screenshot_2023-01-22-20-04-22-673_org joinmastodon android sk debug-edit](https://user-images.githubusercontent.com/6217438/213934803-5184b135-a4cc-46ee-847a-faee2c627e2c.jpg)

already implemented for posting (will be out with the next release!), but thanks for reminding me about boosting, i totally forgot about that

**Comment by sk22** [comment]
also, i didn't even take a look at how glitch-soc is doing it, so i find it pretty funny that it's even worded almost identically :D

**Comment by sk22** [comment]
actually, i think i'd rather have an indicator for alt text-less posts than a confirmation dialog for boosts. there will be ALT-indicators for images which have an alt text in the next release (this got implemented in the official app), and i'd like to enhance that to display an ❗ instead when an image doesn't have an alt text. what do you think about this?

**Comment by deanveloper** [comment]
I think that'd be a good solution! As long as there's something warning me in both cases.
